knowledge-base/grid-custom-grouping-order.md

```diff
@@ -0,0 +1,188 @@
+---
+title: Custom Grouping Order in Grid for Blazor
+description: Learn how to implement custom grouping order in the Telerik Grid for Blazor.
+type: how-to
+page_title: Implementing Custom Grouping Order in Blazor Grid
```

Comment:
Consider a greater variety of keywords, for example use the phrase "sort the groups" in the description. I am also not sure whether it's better to use "group order" or "grouping order", but again, you can use both.